src/style.css

```diff
@@ -1 +1,65 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--0 {
+  background-image: url(/src/images/star.svg);
+  width: 13px;
+  height: 12px;
+}
+
+.stars--1 {
+  background-image: url(/src/images/star.svg);
+  width: 13px;
+  height: 12px;
+}
+
+.stars--2 {
+  background-image: url(/src/images/star.svg);
+  width: 13px;
+  height: 12px;
+}
+
+.stars--3 {
+  background-image: url(/src/images/star.svg);
+  width: 13px;
+  height: 12px;
+}
+
+.stars--4 {
+  background-image: url(/src/images/star.svg);
+  width: 13px;
+  height: 12px;
+}
+
+.stars--5 {
+  background-image: url(/src/images/star-active.svg);
+  width: 13px;
+  height: 12px;
+}
```

Comment:
I created a new branch "developer" because I called this one "stars". I apologize, I'll be more careful next time. Here are links to the final result and the pull request:

https://github.com/mate-academy/layout_stars/pull/3872
https://kirillstetsenko.github.io/layout_stars/
https://kirillstetsenko.github.io/layout_stars/report/html_report/